src/utilities/getComputedStyleValue.ts

```diff
@@ -0,0 +1,22 @@
+// https://stackoverflow.com/a/62557418/3511012
+function globalExists(varName: string): boolean {
+  const globalEval = eval
+  try {
+    globalEval(varName)
+    return true
+  } catch {
+    return false
+  }
+}
+
+export function getComputedStyleValue(element: Element, property: string): string | undefined {
```

Comment:
Can we make this another composition? `useElementComputedStyle` That could be used like this could be really useful. 

```typescript
const { paddingLeft, paddingRight } = useElementComputedStyle(elementRef)
```

Reply:
love this